nupic/research/frameworks/vernon/trainables.py

```diff
@@ -340,18 +330,8 @@ class ContinualLearningTrainable(SupervisedTrainable):
     # This is used in the run_with_raytune function to clarify the stopping condition.
     stop_condition = "num_tasks"
```

Comment:
Can't we move this to the Experiment class instead, and ditch this Trainable as well? This can be an attribute or a function, I think an attribute makes more sense.

Reply:
I added that, although I'm not sure it's needed. Isn't the stop condition adequately covered by `should_stop`?

Reply:
Ok, I got rid of this. Yeah, it's covered by `should_stop`. If we want to make future changes to the stopping logic, we can. This change was more of a refactor, I didn't put much thought into what is the best way to specify stopping criteria.